packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -214,7 +215,10 @@ const errorsContent = (
 
 const overviewContent = (
   <Grid container spacing={3} alignItems="stretch">
-    <Grid item md={6}>
+    <Grid item xs={12}>
```

Comment:
That's a very good catch. I've added an EntitySwitch. 

Do you think the conditional rendering inside `EntityOrphanWarning` should go away entirely now that there's an `isOrphan` method available?

Reply:
Probably. The whole thing is a bit unfortunate since we need to put a grid item inside - which means we can't self-contain it - which means the amount of code balloons into a bunch of lines and two imports, just to get the functionality in.

Did you double check whether any vertical space was being taken if the thing wasn't showing? If it doesn't then let's not change things :) But if it does, then yeah, maybe the check should not go inside the inner component